Web/Admin.UI/src/app/components/core/vd-icon/vd-icon.component.html

```
@@ -0,0 +1,16 @@
+<i class="vd-icon" [ngClass]="class">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)